js/dvfTables.js

```diff
@@ -7,7 +7,7 @@
    */
   Drupal.behaviors.dvfTables = {
     attach: function (context) {
-      $('[data-dvftables]', context).once('dvf-tables').each(function () {
+      $(once('dvf-tables', '[data-dvftables]', context)).each(function () {
```

Comment:
@gitmonkee I don't think this is the correct usage of `once` see https://www.drupal.org/docs/drupal-apis/javascript-api/javascript-api-overview#s-example-using-drupalbehaviors

```
   once('dvfTables', '[data-dvftables]', context).forEach(
      function (element) {
        $table = $(element);
        ...
      }
    );
```

Obviously still works but not sure if it is optimal